README.md

```diff
@@ -26,6 +26,7 @@ In this theme you can add variables to your site config file. The following is t
 	  twitter   = "https://twitter.com/rakuishi07"
 	  facebook  = "https://www.facebook.com/ochiishikoichiro"
 	  github    = "https://github.com/rakuishi/"
+	  gitlab    = "https://gitlab.com/rakuishi/"
```

Comment:
I understand this link is invalid, but I thought it would be fine given it is just an example.